types/staticmaps/index.d.ts

```diff
@@ -26,15 +26,35 @@ declare class StaticMapsImage {
     buffer: (mime?: string, outputOptions?: OutputOptions | PngOptions | JpegOptions | WebpOptions) => Promise<Buffer>;
 }
 
+export type ZoomRange = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17;
```

Comment:
For our custom tiles we go up to 23. Is there a large advantage of using this union over the general `number` type?
The naming of the option `zoomRange` is to indicate a range (min and max) of zoomlevels so as a minor comment this type alias should be `ZoomLevel`

Reply:
> is there a large advantage of using this union over the general `number` type?

There are at least two, intellisense and keeping in the range, but that's disputable. I can amend this and just use number (and correct JSDoc). Just let me know

![image](https://user-images.githubusercontent.com/14539/82941760-3669c180-9f97-11ea-9e85-b7b2c7b209b4.png)
> Type '25' is not assignable to type '1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | undefined'.



> The naming of the option `zoomRange` is to indicate a range (min and max) of zoomlevels so as a minor comment this type alias should be `ZoomLevel`

amended, thanks!

Reply:
Okay I see, the union looks good 👍 Thanks!